game/game_application.cpp

```diff
@@ -22,12 +37,19 @@ Ref<Application> CreateRootexApplication()
 Optional<String> GameApplication::getSceneNameFromCommandLine(const char* s)
 {
 	String cmdLine = s;
-	size_t found = cmdLine.find("game/assets/");
-	if (found != String::npos)
+	if (checkReservedName(cmdLine))
+	{
+		size_t found = cmdLine.find("game/assets/");
+		if (found != String::npos)
+		{
+			return cmdLine.substr(found, cmdLine.size() - 1);
+		}
+		return {};
+	}
+	else
 	{
-		return cmdLine.substr(found, cmdLine.size() - 1);
+		return {};
 	}
-	return {};
 }
```

Comment:
<pre><code>    }
    return  {};
}
</code></pre>  
will be logically equivalent

Reply:
done